subtests/docker_cli/run_flags/run_flags.py

```diff
@@ -0,0 +1,94 @@
+"""
+Summary
+-------
+Checks that docker daemon is running with expected command-line options.
+
+Operational Summary
+-------------------
+Get the full command line of the currently-running docker or docker-latest
+daemon. Check for presence of expected option flags.
+
+"""
+
+import os
+from distutils.version import LooseVersion   # pylint: disable=E0611,F0401
```

Comment:
Hmmm, guess I'm wrong.

BTW, you can use ``git commit --fixup <ID> /path/to/file`` and it'll do the right thing so ``git rebase -i --autosquash`` works later.